inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -98,12 +98,83 @@ def article_exists(obj, eng):
     """
     matches = dedupe_list(match(obj.data))
```

Comment:
I was sloppy here, and relied on the default configuration of the matcher, but it should be passed as a parameter (fetched from the configuration, as observed in https://github.com/inspirehep/inspire-next/pull/3129/files#r171278799).